include/pfs/drp/stella/Controls.h

```diff
@@ -3,6 +3,8 @@
 
 #include <vector>
 #include "lsst/base.h"
+#include "lsst/pex/config.h"
+#include "ndarray.h"
```

Comment:
The only explanation I have for some of the comments is that something went wrong during a rebase. I have fixed these commits again and compiled each one of them